src/plugin.video.icdrama/addon.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.icdrama" version="2.2.0" name="Icdrama" provider-name="aznhusband, dumbo0001, s_h, boleanly, mugol, General-Meow, cthlo">
+<addon id="plugin.video.icdrama" version="2.2.1+matrix.1" name="Icdrama" provider-name="aznhusband, dumbo0001, s_h, boleanly, mugol, General-Meow, cthlo">
   <requires>
-    <import addon="xbmc.python" version="2.1.0"/>
-    <import addon="script.module.html5lib" version="0.999.0"/>
-    <import addon="script.module.beautifulsoup4" version="4.3.2"/>
-    <import addon="script.module.requests" version="2.9.1"/>
-    <import addon="script.module.resolveurl" version="5.0.38"/>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.html5lib" version="1.0.1+matrix.1"/>
+    <import addon="script.module.beautifulsoup4" version="4.9.3+matrix.1"/>
+    <import addon="script.module.requests" version="2.22.0+matrix.1"/>
+    <import addon="script.module.resolveurl" version="5.1.25"/>
```

Comment:
if `script.module.resolveurl` doesn't come with a version (number) that can be used with Matrix without manual changes then it will be impossible for the icdrama addon to install on Kodi 19.

Workaround is to take a copy the `script.module.resolveurl`, modify the addon.xml to include the `import` line and host it in this repository.
Con: Need to do this for every new release.

Reply:
Yea that could work but we will need to package a new modified `resolveurl` whenever we want to increment the version number. Perhaps we can ask the maintainer of `resolveurl` to release a Matrix version?

Reply:
I submitted an issue with the question: https://github.com/jsergio123/script.module.resolveurl/issues/433. Now we wait...

Reply:
I think resolveurl is actually Py 2/3 compatible.

`<import addon="xbmc.python" version="3.0.0" />`
This isn't actually unnecessary for their script to install/run in Kodi 18/19. I was able to sideload both resolveurl and icdrama without the import python line in both Kodi 18 and 19. But I'm not sure if it works correctly in the repo, and it'll probably might not work in the future

Reply:
Hmm what do you mean with sideload? install the addon zip file in Kodi? I will give it a try later
I thought you would get a dependency error and installation will fail.
They probably won't fix it then..

Reply:
Manual installing zip file works. Still trying to figure out how to install icdrama on fresh Kodi 19. It doesn't seem to install the dependencies (e.g. bs4).

Reply:
Never mind my previous comment. I can install the icdrama addon using this pull request even without version changes in the addon.xml. I currently have it like:
```xml
<requires>
    <import addon="script.module.html5lib" version="0.999.0"/>
    <import addon="script.module.beautifulsoup4" version="4.3.2"/>
    <import addon="script.module.requests" version="2.9.1"/>
    <import addon="script.module.resolveurl" version="5.0.38"/>
  </requires>
```
In Kodi 19 it automaticall get the Matrix version.
For ResolveURL I get the version of gujal `5.1.26`.

I am using a clean Kodi 19. Will continue checking if it works when integrated with a repository.